frontend/src/main/pages/Moderate.js

```diff
@@ -2,10 +2,19 @@ import React from "react";
 import { useCurrentUser, hasRole } from "main/utils/currentUser";
```

Comment:
This file should be called ModeratePage.js to follow the naming convention throughout all of the apps we've worked on in this course.